nadlogar/problems/models/meta.py

```diff
@@ -47,13 +47,15 @@ class GeneratedDataIncorrect(Exception):
 
 
 class Problem(models.Model):
+    default_instruction = None
+    default_solution = None
     document = models.ForeignKey("documents.Document", on_delete=models.CASCADE)
     content_type = models.ForeignKey(
         ContentType,
         on_delete=models.PROTECT,
         limit_choices_to=limit_content_type_choices,
     )
-    text = models.ForeignKey("problems.ProblemText", on_delete=models.PROTECT)
```

Comment:
Zakaj smo tukaj spremenili iz PROTECT na SET_NULL, in zakaj smo dodali argumenta blank=True, null=True?

Reply:
Možnost, da se pri problemu uporabi (dinamično) privzeto besedilo predstavimo tako, da je atribut `text` nastavljen na `None`. Prej je bil to obvezen argument, z `blank=True, null=True` ga naredimo neobveznega (oba sta potrebna, ker tako rabi Django). Prej je `PROTECT` pomenil, da objekta `ProblemText` nismo smeli pobrisati, če je obstajal kakšen `Problem`, ki ga je uporabljal. Če imamo `SET_NULL`, se ustrezni `text` nastavi na `None`, če pobrišemo `ProblemText`.